src/App.js

```diff
@@ -1,29 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import React from 'react';
+import './styles.css';
+import Footer from './components/Footer/index'
+import Header from './components/Header/index'
+import HomePage from './components/HomePage';
+
 
 function App() {
-  const { data, isLoading } = useFeaturedBanners();
-  console.log(data, isLoading);
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
```

Comment:
When no styling or specific purpose, is preferred to use React.Fragment instead, `<></>`